packages/code-editor/src/index.ts

```diff
@@ -1,17 +1 @@
-export {
-  CodeEditor,
-  type CodeEditorProps,
-  CodeEditorSelectors,
-  type CodeEditorTooltip,
-  CodeEditorTooltipSeverity,
-  type CodeMirrorExtension,
-  type CodeMirrorRef,
-  type CodeMirrorState,
-  type CodeMirrorView,
-  createCodeMirrorHighlightExtension,
-  createCodeMirrorLanguageExtension,
-  createCodeMirrorThemeExtension,
-  createCodeMirrorTooltipsExtension,
-  IndentUnits,
-  LanguageName,
-} from './CodeEditor';
+export * from './CodeEditor';
```

Comment:
just a note on this, I find it easier to debug when I explicitly name all the exports (in case I forgot to export something from the `index` one level down)